Marksman/Compl.fs

```diff
@@ -639,8 +639,6 @@ module Completions =
             Some
                 { CompletionItem.Create(label) with
                     Detail = Some detail
-                    // Use input as filter text to avoid any extra filtering on the editor's side
```

Comment:
@TheBlob42 did this line cause problems?

Reply:
Exactly. The `FilterText` property is the string that the user input is matched against when filtering the completion candidates (see [here](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#completionItem)). If you set this value for example to `Some "foo"` and trigger the completion then typing "foo" in your editor should still show all completion candidates (this is an extreme example I know).

The problem here was that the filter text was set to the user input at the time of the completion candidates creation. Therefore every "additional" character would not match this text anymore and the completion candidates would be gone.

Reply:
Thanks for the fix here! I think there's a _same-ish_ problem with other types of completions... I haven't fully figured out the deal with client side filtering of completion candidates.